lib/galaxy_test/selenium/test_workflow_editor.py

```diff
@@ -671,8 +668,8 @@ def workflow_editor_maximize_center_pane(self, collapse_left=True, collapse_righ
 
     def workflow_editor_connect(self, source, sink, screenshot_partial=None):
         source_id, sink_id = self.workflow_editor_source_sink_terminal_ids(source, sink)
-        source_element = self.driver.find_element_by_css_selector(f"#{source_id}")
-        sink_element = self.driver.find_element_by_css_selector(f"#{sink_id}")
+        source_element = self.driver.find_element(By.CSS_SELECTOR, f"#{source_id}")
+        sink_element = self.driver.find_element(By.CSS_SELECTOR, f"#{sink_id}")
```

Comment:
Did you want to also replace:

```suggestion
        source_element = self.driver.find_element(self.by.CSS_SELECTOR, f"#{source_id}")
        sink_element = self.driver.find_element(self.by.CSS_SELECTOR, f"#{sink_id}")
```

? There are still a few around.